packages/mobile/src/home/HomeTokenBalance.tsx

```diff
@@ -15,10 +15,14 @@ import ProgressArrow from 'src/icons/ProgressArrow'
 import { getLocalCurrencySymbol } from 'src/localCurrency/selectors'
 import { navigate } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
-import { tokensWithBalanceSelector, totalTokenBalanceSelector } from 'src/tokens/selectors'
+import {
+  tokensWithTokenBalanceSelector,
+  tokensWithUsdValueSelector,
+  totalTokenBalanceSelector,
+} from 'src/tokens/selectors'
 
 function TokenBalance() {
-  const tokenBalances = useSelector(tokensWithBalanceSelector)
+  const tokenBalances = useSelector(tokensWithUsdValueSelector)
```

Comment:
nit: Should we rename the variable too?
```suggestion
  const tokensWithUsdValue = useSelector(tokensWithUsdValueSelector)
```